docs/SAFE-SULONG.md

```diff
@@ -0,0 +1,218 @@
+Safe Sulong is a version of Sulong that only uses managed memory (Java allocations)
+to prevent temporal and spatial memory safety errors. You can read about
+its architecture in
+[Sulong: Memory Safe and Efficient Execution of LLVM-Based Languages](http://ssw.jku.at/General/Staff/ManuelRigger/ECOOP16-DS.pdf).
+While also other tools (such as Clang's AddressSanitizer) can find memory
+errors, Sulong strives to provide memory safety "per default", since not
+all developers use static analysis tools, while not sacrificing peak performance.
```

Comment:
I removed the peak performance aspect, since we do not have numbers yet to provide it.